prow/config.yaml

```diff
@@ -99,12 +99,25 @@ branch-protection:
           branches: *blocked_branches
         operator:
           branches: *blocked_branches
+            master:
+              required_status_checks:
+                contexts:
+                  - operator-lint-master
+                  - operator-build-master
+                  - operator-test-master
+                  - operator-mandiff-master
         cni:
           branches: *blocked_branches
         installer:
           branches: *blocked_branches
         pkg:
           branches: *blocked_branches
+            master:
+              required_status_checks:
+                contexts:
+                  - pkg-lint-master
```

Comment:
you shouldn't need these. By adding the config its already required. If you want to make a test not required you set `optional: true` on the job

Reply:
Yeah right now this PR seems unnecessary. We only need to define these for foreign contexts that prow does not manage -- like CircleCI's or cla: yes.

Prow job contexts should not go in here as they are added automatically